### ISSUE TITLE
Allow for binaries without a file extension

### ISSUE THREAD

**Issue by A-dead-pixel** [MikolajKolek/toster#20]
Imo it would be nice if I could just specify a program binary to execute instead of giving `toster` a source file.

**Comment by MikolajKolek** [comment]
Actually you can! Just give the path to an executable file instead of a source file, it's even stated in the help page and README

**Comment by A-dead-pixel** [comment]
As I encountered a crash when giving it the binary, I just assumed it wasn't supported.
I'm guessing from the traceback that the lack of proper handling of files without extensions is at fault.

```
name = "toster"
operating_system = "Artix Linux [64-bit]"
crate_version = "1.1.3"
explanation = """
Panic occurred in file '/home/pixl/.cargo/registry/src/index.crates.io-6f17d22bba15001f/toster-1.1.3/src/main.rs' at
 line 241
"""
cause = "Couldn't get the extension of the provided file"
method = "Panic"
backtrace = """

   0: 0x561ae175bca3 - core::panicking::panic_display::h53b76d6a5b36059a
                at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/core/src/panicking.rs:150
                 - core::panicking::panic_str::h27c52b3b276cc44c
                at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/core/src/panicking.rs:134
                 - core::option::expect_failed::h4aa782639484e720
                at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/core/src/option.rs:1988
   1: 0x561ae17619ea - toster::main::h134a69fa54e03ab9
   2: 0x561ae176fc23 - std::sys_common::backtrace::__rust_begin_short_backtrace::h4af996e398c3c7b3
   3: 0x561ae1781239 - std::rt::lang_start::{{closure}}::h1bc742a096ab222d
   4: 0x561ae18778bb - core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once::h9ac99ef
7f6c784c8
                at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/core/src/ops/function.rs:284
                 - std::panicking::try::do_call::h0f22c2589be0f48c
                at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/panicking.rs:502
                 - std::panicking::try::hd7c132f6868fcd47
                at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/panicking.rs:466
                 - std::panic::catch_unwind::hacba6bf8c762a298
                at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/panic.rs:142
                 - std::rt::lang_start_internal::{{closure}}::h7cc864010c3e2dc6
                at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/rt.rs:148
                 - std::panicking::try::do_call::h842528acce04f27a
                at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/panicking.rs:502
                 - std::panicking::try::h6c6444102842c8af
                at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/panicking.rs:466
                 - std::panic::catch_unwind::h6d567c7fc72669f6
                at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/panic.rs:142
                 - std::rt::lang_start_internal::hc2c145acab4a2a95
                at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/rt.rs:148
   5: 0x561ae1762325 - main
   6: 0x7f23ddfd7cd0 - <unresolved>
   7: 0x7f23ddfd7d8a - __libc_start_main
   8: 0x561ae175c405 - _start
   9:        0x0 - <unresolved>"""
```

**Comment by MikolajKolek** [comment]
Ah sorry, yes that might be a problem, I'll fix that soon

**Comment by MikolajKolek** [comment]
@A-dead-pixel The issue is fixed in Toster 1.1.4

**Comment by A-dead-pixel** [comment]
Yeah, it works great now, thanks!
